go.mod

```diff
@@ -6,15 +6,15 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/hashicorp/go-version v1.6.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.29.0
-	github.com/rancher/rke v1.4.10
+	github.com/rancher/rke v1.4.10-rc1
```

Comment:
ugh, going back to an older version again?  The robot needs to be fixed.

Reply:
Thanks, I'll close this issue and file an issue for dependabot to be fixed.